pandas/tests/scalar/timedelta/test_arithmetic.py

```diff
@@ -328,6 +336,16 @@ def test_td_div_numeric_scalar(self):
         assert isinstance(result, Timedelta)
         assert result == Timedelta(days=2)
 
+    @pytest.mark.parametrize('nan', [np.nan, np.float64('NaN'), float('nan')])
```

Comment:
nulls_fixture does this

Reply:
Yes, but it also includes several scalars we _dont_ want to include here.